doc/specs/stdlib_stats_distribution_exponential.md

```diff
@@ -0,0 +1,243 @@
+---
+title: stats_distribution_exponential
+---
+
+# Statistical Distributions -- Exponential Distribution Module
+
+[TOC]
+
+## `rvs_expon` - exponential distribution random variates
+
+### Status
+
+Experimental
+
+### Description
+
+An exponentially distributed random variate distribution is the distribution of time between events in a Poisson point process. The inverse scale parameter `lambda` specifies the rate of change.
+
+Without argument the function returns a standard exponential distributed random variate E(1) with `lambda = 1`.
```

Comment:
Consider rephrasing as:
* Without argument the function returns a random sample from the standard exponential distribution `E(1)` with `lambda=1`